jumpy/lax.py

```diff
@@ -2,21 +2,32 @@
 
 from __future__ import annotations
 
-from typing import Any, Callable, TypeVar
+from typing import TYPE_CHECKING, Any, Callable, Sequence, TypeVar
 
 import numpy as onp
 
-from jumpy import is_jax_installed, ndarray
+from jumpy import is_jax_installed
 from jumpy.core import is_jitted, which_np
 
+if TYPE_CHECKING:
+    from jumpy.numpy import ndarray
+
 try:
     import jax
     import jax.numpy as jnp
 except ImportError:
     jax, jnp = None, None
 
 
-__all__ = ["cond", "fori_loop", "scan", "top_k", "while_loop"]
+__all__ = [
+    "cond",
+    "switch",
+    "fori_loop",
+    "scan",
```

Comment:
Could the functions be in alphabetical order?

Reply:
yes, will do!